### PR TITLE
Updates footer components

### PR DESCRIPTION
**Pull request by summer-cook**
## Summary
- makes the footer stick to the bottom
- provides a second option for footer component (copied from bootstrap)
- makes original footer responsive
- adds an optional height prop for the logo

## Related
https://github.com/scientist-softserv/webstore-component-library/issues/73

## Screenshots

OG Footer:
image
image
image

Bootstrap footer:
image
image

## Video
https://share.getcloudapp.com/yAuAomqL

### CODE REVIEW

**Comment by alishaevn** [comment]
why not replace the Footer with the bootstrap version instead of having 2?

also, is there a reason that the logo is centered in the bootstrap version instead of being left aligned? and what how do the link groups display in that version?

**Comment by summer-cook** [comment]
@alishaevn I left both versions for more options from the user side- the bootstrap one is a totally different layout than your version, so i figured might as well keep 2 options.

I also updated your version to use bootstrap classes & be more responsive, so that either one would be a viable option to switch out in app.js in the webstore. 

they are really close to each other, but depending on what the client likes they can choose between either layout. 
the links in the bootstrap version are just an array of links rather than an array of objects like the link groups, because they are just a single group and don't have a heading so they really only needed to be an array

File: src/compounds/Footer/Footer2.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
```
Comment (alishaevn) [inline src/compounds/Footer/Footer2.stories.jsx:1]: I'm curious about whether we can refactor this file into "Footer.stories.jsx"...but we can save that for another day. 🙂

**Review by alishaevn — APPROVED** [review]

File: src/compounds/Footer/Footer2.stories.jsx (hunk 1 above)
Comment (summer-cook) [reply src/compounds/Footer/Footer2.stories.jsx:1]: ooooh thats a good point! i think that could potentially be doable

**Pull request merged**